src/test/java/ro/msg/learning/shop/testcontainer/TestContainerTest.java

```diff
@@ -0,0 +1,54 @@
+package ro.msg.learning.shop.testcontainer;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import ro.msg.learning.shop.IntegrationTest;
+import ro.msg.learning.shop.model.Address;
+import ro.msg.learning.shop.model.Location;
+import ro.msg.learning.shop.repository.LocationRepository;
+
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class TestContainerTest extends IntegrationTest {
```

Comment:
What is this test for?

Reply:
I renamed it to IsolationTest to reflect on what it does: it verifies that each test is isolated by checking whether the location table is empty, saving a new location, and then in the second test checking that the table is empty again.

Reply:
removed it